packaging/etc/engine-config/engine-config.properties

```diff
@@ -550,6 +550,9 @@ LiveSnapshotPerformFreezeInEngine.type=Boolean
 LiveSnapshotFreezeTimeout.description=The live snapshot command without memory timeout in minutes
 LiveSnapshotFreezeTimeout.type=Integer
 LiveSnapshotFreezeTimeout.validValues=1..3000
+VolumeUtilizationChunkInMB.description="Size of extension chunk in megabytes. Use higher values to extend in bigger chunks.."
```

Comment:
Double .. at the end - is this intended? looks like an error.

Reply:
Done.
Removed the extra "`.`", thanks.
Also will move the whole commit to a separate PR as we discussed.